.github/workflows/realease-on-pr-merge.yaml

```
@@ -39,17 +39,25 @@ jobs:
           elif [ "$BUMP_TYPE" = "minor" ]; then
-            npm version minor
+            NEW_TAG=$(npm version minor)
           elif [ "$BUMP_TYPE" = "patch" ]; then
-            npm version patch
+            NEW_TAG=$(npm version patch)
           fi
+          
+          # Extract tag from npm version output
+          NEW_TAG=$(echo $NEW_TAG | tr -d 'v')
+          echo "new_tag=$NEW_TAG" >> $GITHUB_OUTPUT
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)